src/test/java/com/emc/object/util/TestProperties.java

```diff
@@ -32,6 +32,7 @@
     public static final String S3_ENDPOINT = "s3.endpoint";
     public static final String S3_ACCESS_KEY = "s3.access_key";
     public static final String S3_SECRET_KEY = "s3.secret_key";
+    public static final String S3_SECURITY_TOKEN = "s3.security_token";
 
```

Comment:
I would add a separate set of properties for temporary credentials:
```
s3.temp_access_key
s3.temp_secret_key
s3.temp_security_token
```

Reply:
done